docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      NODE_ENV: 'development'
+    restart: unless-stopped
+    command: npm start
+    image: zlesyk/hackathon-client:latest
+    container_name: hackathon-client
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/app
+    depends_on:
+      - server
+  server:
```

Comment:
Does this setup (overall) work? I.e. can you locally start the container(s) and it works?

Reply:
it works)))